french/maps/outback_02.rpy

```diff
@@ -1,314 +1,313 @@
 
 # game/maps/outback_02.rpy:358
 translate french outback_talk_04_5e568321:
 
     # e nulla "He was...{w=0.5} Used to spoil me rotten, really.{w=0.3} Me and Andrea both."
-    e nulla ""
+    e nulla "Il l'était...{w=0.5} Il avait l'habitude de me gâtter pourri, vraiment.{w=0.3} Andrea et moi tous les deux."
 
 # game/maps/outback_02.rpy:362
 translate french outback_talk_04_bb17f93f:
 
     # j nulla "Oh...{w=0.5} He was Andrea's father?"
-    j nulla ""
+    j nulla "Oh...{w=0.5} Il était le père d'Andrea ?"
 
 # game/maps/outback_02.rpy:364
 translate french outback_talk_04_4c8640bb:
 
     # e nulla "Yeah...{w=0.5} Let's just say that they may have used a little too much carrot and not enough stick with that one."
-    e nulla ""
+    e nulla "Ouais...{w=0.5} Disons simplement qu'ils ont peut-être un peu trop utilisé la carotte et pas assez le bâton avec celui-là."
 
 # game/maps/outback_02.rpy:366
 translate french outback_talk_04_2e1649dd:
 
     # e nulla "No pun intended."
-    e nulla ""
+    e nulla "Sans mauvais jeu de mot."
 
 # game/maps/outback_02.rpy:368
 translate french outback_talk_04_ac282c96:
 
     # j nulla "Pffft-!"
-    j nulla ""
+    j nulla "Pffft- !"
 
 # game/maps/outback_02.rpy:382
 translate french outback_talk_05_45d063fe:
 
     # e amused "Soooo, are you going to talk or do I have to think it's a stick-up?"
-    e amused ""
+    e amused "Alooors, tu as quelque chose à dire ou est-ce que je dois penser que c'est un braquage ?"
 
 # game/maps/outback_02.rpy:383
 translate french outback_talk_05_25623450:
 
     # ru laugh blush g_up "Eeeeek!{w=0.3} Sorry, sorry, so sorry!"
-    ru laugh blush g_up ""
+    ru laugh blush g_up "Eeeeek !{w=0.3} Désolé, désolé, je suis si désolé !"
 
 # game/maps/outback_02.rpy:390
 translate french outback_talk_05_8375e729:
 
     # ru nulla "I just wanted to thank you...{w=0.5} But I don't even know how to put it into words."
-    ru nulla ""
+    ru nulla "Je voulais juste te remercier...{w=0.5} Mais je ne sais même pas comment le dire avec mes mots."
 
 # game/maps/outback_02.rpy:391
 translate french outback_talk_05_54062f2d:
 
     # e nulla "Oh, shut up.{w=0.3} You're the one who did everything."
-    e nulla ""
+    e nulla "Oh, tais-toi.{w=0.3} C'est toi qui a tout fait."
 
 # game/maps/outback_02.rpy:393
 translate french outback_talk_05_520d9407:
 
     # ru nulla "But you're the one who pushed me to do it...{w=0.5} And I couldn't be happier I did."
-    ru nulla ""
+    ru nulla "Mais tu es celle qui m'a poussé à le faire...{w=0.5} Et Je ne pourrais pas être plus heureuse."
 
 # game/maps/outback_02.rpy:396
 translate french outback_talk_05_82dfbefd:
 
     # ru nulla "I'm still a bit conflicted about the whole thing but...{w=0.5} Now I have the opportunity to unravel that mess, you know?"
-    ru nulla ""
+    ru nulla "Je reste un peu confuse à propose de tout le truc mais...{w=0.5} Maintenant j'ai l'opportunité de démêler ce désordre, tu sais ?"
```

Comment:
à propos de *
conflicted -> partagé (je reste un peu partagée)

you know -> tu comprends (c'est une histoire de contexte en anglais qui change le sens)